File: Discord Webhook/ViewController.m
//
//  ViewController.m
//  Discord Webhook
//
//  Created by Cian on 11/10/17.
//  Copyright © 2017 Chun Industries. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

@synthesize urlLabel;
@synthesize usernameLabel;
@synthesize iconURLLabel;
@synthesize contentLabel;

-(BOOL) textFieldShouldReturn:(UITextField *)urlLabel{
        [urlLabel resignFirstResponder];
        return YES;
}
-(BOOL) textFieldShouldReturn1:(UITextField *)usernameLabel{
        [urlLabel resignFirstResponder];
        return YES;
}
-(BOOL) textFieldShouldReturn2:(UITextField *)iconURLLabel{
        [urlLabel resignFirstResponder];
        return YES;
}
-(BOOL) textFieldShouldReturn3:(UITextField *)contentLabel{
        [urlLabel resignFirstResponder];
        return YES;
}



- (void)viewDidLoad {
    // Do any additional setup after loading the view, typically from a nib.
    
    [urlLabel setDelegate:self];
    [usernameLabel setDelegate:self];
    [iconURLLabel setDelegate:self];
    [contentLabel setDelegate:self];

    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: DiscordWebhook2/ViewController.swift
//
//  ViewController.swift
//  DiscordWebhook2
//
//  Created by Cian on 8/1/18.
//  Copyright © 2018 SpaceBar Incorporated. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var txtWebhookURL: UITextField!
    @IBOutlet weak var txtContent: UITextField!
    @IBOutlet weak var txtUsername: UITextField!
    @IBOutlet weak var txtProfilePic: UITextField!
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func buSend(_ sender: Any) {
    //activated when button clicked
    let webhookURL = txtWebhookURL.text!
    let content = txtContent.text!
    let userName = txtUsername.text!
    let profilePic = txtProfilePic.text!
    // let msgJSON =
    
        
    }
    
}

